katalog/istio/README.md

```diff
@@ -17,169 +34,175 @@ Source: [https://istio.io/docs](https://istio.io/docs/concepts/what-is-istio/#wh
-  internally
-  - **Sidecars:** You can use a sidecar configuration to do the following:
-    - Fine-tune the set of ports and protocols that an Envoy proxy accepts.
-    - Limit the set of services that the Envoy proxy can reach.
-- ***Network resilience and testing:*** Istio provides opt-in failure recovery and fault injection features that you
-can configure dynamically at runtime.
-  - **Timeouts:** A timeout is the amount of time that an Envoy proxy should wait for replies from a given service,
-  ensuring that services don’t hang around waiting for replies indefinitely and that calls succeed or fail within a
-  predictable timeframe.
-  - **Retries:** A retry setting specifies the maximum number of times an Envoy proxy attempts to connect to a service
-  if the initial call fails.
-  - **Circuit breakers:** In a circuit breaker, you set limits for calls to individual hosts within a service, such
-  as the number of concurrent connections or how many times calls to this host have failed.
-  - **Fault injection:** Fault injection is a testing method that introduces errors into a system to ensure that it
-  can withstand and recover from error conditions.
-- ***Policies:*** Istio lets you configure custom policies for your application to enforce rules at runtime such as:
-  - Rate limiting to dynamically limit the traffic to a service
-  - Denials, whitelists, and blacklists, to restrict access to services
-  - Header rewrites and redirects
-- ***Observability:*** Istio generates detailed telemetry for all service communications within a mesh. This telemetry
-provides observability of service behavior, empowering operators to troubleshoot, maintain, and optimize their
-applications – without imposing any additional burdens on service developers.
-
-## Image repository and tag
-
-All istio container images are currently available at dockerhub: [docker.io/istio](https://hub.docker.com/u/istio)
-
-* istio container images: `docker.io/istio/*`
-
-Includes:
-
-- docker.io/istio/proxyv2
-- docker.io/istio/mixer
-- docker.io/istio/pilot
-
-* addons container images:
-
-- Kiali: quay.io/kiali/kiali:v1.9
-
-## Deployment
-
-You can deploy istio by running following command in the root of the project:
-
-```shell
-$ kustomize build | kubectl apply -f -
+In order to install the Istio package, it's required to have installed the [init](./init/README.md) package.
+This is the one involved in the Istio CRDs creation.
+
+```bash
+$ kustomize build katalog/istio/init | kubectl apply -f -
+$ kubectl -n istio-system wait --for=condition=complete job --all
+job.batch/istio-init-crd-10-1.4.2 condition met
+job.batch/istio-init-crd-11-1.4.2 condition met
+job.batch/istio-init-crd-14-1.4.2 condition met
 ```
 
-### Configuration
+Once installed, the cluster will be ready to install the [minimal](#minimal-installation) Istio deployment.
 
-#### Kiali
 
-##### Admin user
+## Modular
```

Comment:
change Modular -> Modules?

Reply:
With `modular` i wanted to say the design was modular, i did not pretend to list `modules`.

Reply:
Maybe changing from `Modular` -> `Modular design`

Reply:
ah, ok! `Modular design` is better then in my opinion